packages/fiori/src/fcl-utils/FCLLayout.ts

```diff
@@ -1,34 +1,43 @@
-const getLayoutsByMedia = () => {
+import { MEDIA } from "../FlexibleColumnLayout.js";
```

Comment:
**import type { MEDIA } from "../FlexibleColumnLayout.js";** here in fcl-utils/FCLLayout

**and export type { MEDIA };** in FlexibleColumnLayout